OracleTuxedo/core/samples/shm/simpapp_runme.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/sh
```

Comment:
If you intend this script to be run by `bash`, you need to change this to `#!/bin/bash`.